### DISCUSSION TITLE
Database error

### DISCUSSION THREAD

**Discussion by cinewar** [blitz-js/blitz#2158] [Q&A]
I really appreciate blitz js. It is perfect. I have one problem. I used aws rds for database. It worked locally and even I built it worked. But when I deployed it to vercel It give an error and did not work. I wonder it was an problem with aws rds?

**Comment by flybayer** [comment]
The problem is using RDS with serverless (Vercel). I highly recommend to not deploy to Vercel. People have too many problems. :/

For more details, see https://blitzjs.com/docs/deploy-vercel
